app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -716,9 +716,6 @@ else if(resultCode == RESULT_OK) {
                     break;
                 } else if(resultCode == RESULT_OK) {
                     currentState.getSession().setDatum(currentState.getSession().getNeededDatum().getDataId(), intent.getStringExtra(SessionFrame.STATE_DATUM_VAL));
-                    if(intent.hasExtra(CallOutActivity.CALL_DURATION)) {
-                        platform.setCallDuration(intent.getLongExtra(CallOutActivity.CALL_DURATION, 0));
-                    }
```

Comment:
@dcluna Actually, that line calls AndroidCommCarePlatform.setCallDuration, which sets a class field that is never accessed. That is why I took all this out. The functionality seems to have been moved to CalloutActivity.java. Please double check this (that AndroidCommCarePlatform.getCallDuration is never called), but I think what this PR removes is never used.